import { BaseError } from "./BaseError";


export class NotFoundError extends BaseError {
    constructor(){
        super("Recurso n√£o encontrado", 404)
    }
}